mysql/const.go

```diff
@@ -377,6 +377,16 @@ var DefaultLengthOfTimeFraction = map[int]int{
 // See https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html
 type SQLMode int
 
+// IsNoZeroDateMode detects if 'NO_ZERO_DATE' mode is set in SQLMode
+func (m SQLMode) IsNoZeroDateMode() bool {
```

Comment:
1. s/ IsXXX/ HasXXX
2. add test for this function.

Reply:
Done @XuHuaiyu . PTAL